### PR TITLE
loadout verification tools for #25

### PR DESCRIPTION
**Pull request by McDiod**
for #25 

Adds chat commands `#grad-loadout viewer` and `#grad-loadout verify` plus their respective functions.

## Verifer
Tests all currently loaded loadouts. Does not test all available loadouts from config, because of the number of possible combinations (see https://github.com/gruppe-adler/grad-loadout#classes). Displays errors and warnings in a dialog and writes to rpt.

Checks if:
* items exist
* containers are loaded beyond capacity
* weapon attachments are compatible
* magazines are compatible

## Loadout Viewer
Loadout viewer dialog that displays all items by *container << unit << group << side* in a tree.

Other features:
* cam focussing on selected unit
* info about selected item or container including weight, capacity, space left

**Viewer:**

**Verifier:**

### CODE REVIEW

File: functions/api/fn_verifyLoadouts.sqf [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+    _mass
+};
+
+private _fnc_getLoad = {
+    params ["_container","_itemsList"];
+    _load = 0;
+    {
+        _load = _load + ([_x] call _fnc_getMass);
+        false
+    } count _itemsList;
```
Comment (Fusselwurm) [inline functions/api/fn_verifyLoadouts.sqf:58]: :thinking: warum `count` statt `forEach`

**Review by Fusselwurm — APPROVED** [review]
I dont have time to look as closely at it as I should, but I like the feature :+1:

File: functions/api/fn_verifyLoadouts.sqf (hunk 1 above)
Comment (nomisum) [reply functions/api/fn_verifyLoadouts.sqf:58]: "However, count loop is a little faster than forEach loop, but it does not benefit from the _forEachIndex variable."

**Comment by nomisum** [comment]
Couldnt take an ingame look yet, screens look promising.

File: functions/api/fn_verifyLoadouts.sqf (hunk 1 above)
Comment (McDiod) [reply functions/api/fn_verifyLoadouts.sqf:58]: Hab an der Stelle anscheinend gerade mal dran gedacht - ist nicht konsequent durchgezogen.

Comment (Fusselwurm) [reply functions/api/fn_verifyLoadouts.sqf:58]: ugh, diese bekloppten performance-sachen immer mit sqf… ok :)

**Comment by McDiod** [comment]
Evtl. Bug:
15:43:43 "ERROR: headgear rhssaf_helmet_hgu56p does not exist - B Alpha 3-1:2 (Helikopterpilot)"

Eras sagt, dass der Helm existiert.

**Comment by McDiod** [comment]
I completely forgot about this. This is done, please merge and release if no further questions / issues @Fusselwurm .

**Comment by McDiod** [comment]
There is a test mission on our server `grad-loadout-verificationTest.VR`, so if you ever find a bit of time.

**Comment by McDiod** [comment]
Okay so since this is currently unmaintained, do you still want to review it @nomisum ? Otherwise I'm gonna merge and release.

**Comment by nomisum** [comment]
go ahead. i will try this when its released for sure. @McDiod

**Pull request merged**